pkg/printers/json_test.go

```diff
@@ -0,0 +1,155 @@
+package printers_test
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	. "github.com/weaveworks/eksctl/pkg/printers"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("JSON Printer", func() {
+
+	Describe("When creating New JSON printer", func() {
+		var (
+			printer OutputPrinter
+		)
+
+		BeforeEach(func() {
+			printer = NewJSONPrinter()
+		})
+
+		It("should not be nil", func() {
+			Expect(printer).ShouldNot(BeNil())
+		})
+
+		//TODO - finish
```

Comment:
Does this still need finishing up, or it's rather good enough?

Reply:
Fixed.